mautic/mautic
# Fix date for dynamic contents

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | feature for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      |
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Notice issue with date field in dynamics content




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create custom contact date field
3. Create dynamic content for landing page with filter on date field with not empty operator.
4. Test dynamic content
5. Without PR you always see variant for not empty, even if your field is empty.
6. After PR you should see content properly
